content/warnings/legacy-factories.md

```diff
@@ -27,9 +27,9 @@ function render() {
 }
 ```
 
-## Without JSX {#without-jsx}
+## JSX Olmadan {#without-jsx}
```

Comment:
We can keep this. This describes a scenario. But below, (`Dynamic components without JSX`) it implies a name or an adjective. There is a tiny difference between them.